src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.html

```diff
@@ -26,7 +26,12 @@ <h4>{{ 'ubs-tariffs-add-service.edit_service' | translate }}</h4>
           class="shadow-none form-control"
           id="nameEng"
           placeholder="{{ 'ubs-tariffs-add-service.placeholder-service' | translate }}"
+          maxlength="255"
+          required
         />
+        <div class="error-message" *ngIf="addServiceForm.get('nameEng').invalid && addServiceForm.get('nameEng').touched">
+          {{ 'ubs-tariffs-add-service.error_service_field_message' | translate }}
```

Comment:
Use **app-ubs-input-error** for UBS error messages
For example we use it in **UBSAddAddressPopUpComponent**

Reply:
changes was added